src/components/Editor/MonacoInstance/utils.ts

```diff
@@ -15,18 +15,34 @@ import {setupThemes} from "./theme";
 import {CustomMonacoEditorHandlers} from "./typings";
 
 
+/**
+ * Generates a source name for verifying explicit cursor position changes.
+ *
+ * @param funcName
+ * @param isExplicit
+ * @return
+ */
+const generateExplicitSourceName = (funcName: string, isExplicit: boolean) => {
```

Comment:
let's move this to `MonacoInstance/actions.ts`, which is where the other helper `isSourceExplicit` is exposed